third-party-programs/Velocity-Bench/cudaSift/README.md

```diff
@@ -70,6 +70,7 @@ $ ls compile_commands.json  # make sure compile_commands.json is generated
 compile_commands.json
 ```
 ### 3 Migrate the source code and build script
+## 3.1 Migrate the source code and generate build script `Makefile.dpct`
```

Comment:
add line after L72: 
Follow section 3.1 or section 3.2 to migrate the source code and corresponding build scripts.

Reply:
okay